apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/Fun0ExpectationSamples.kt

```diff
@@ -0,0 +1,83 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class Fun0ExpectationSamples {
+
+    @Test
+    fun toThrowFeature() {
+        expect { throw IllegalStateException("abc") }
+            .toThrow<IllegalStateException>() messageToContain "abc"
+        //      | subject is now of type IllegalStateException
+
+        fails {
+            expect { throw IllegalStateException("abc") }
+                .toThrow<IndexOutOfBoundsException>() messageToContain "abc"
+            // not shown in reporting as `toThrow<IndexOutOfBoundsException>()` already fails
+        }
+    }
+
+    @Test
+    fun toThrow() {
+        expect { throw IllegalStateException("abc") }
+            .toThrow<IllegalStateException> { // subject inside this block is of type IllegalStateException
+                it messageToContain "abc"
```

Comment:
IMO reads a bit nicer but also fine to keep it this way
```suggestion
                its messageToContain "abc"
```